drupal/tpl/values.toml

```diff
@@ -0,0 +1,4 @@
+mariadbPassword = ""
```

Comment:
I think we should change this variable to `mariadbRootPassword` since in the new mariadb image a new parameter is added to set the root password.